Markdown/Symbol List - Heading.tmPreferences

```diff
@@ -4,7 +4,7 @@
 	<key>name</key>
 	<string>Symbol List: Heading</string>
 	<key>scope</key>
-	<string>text.html.markdown markup.heading</string>
+	<string>text.html.markdown markup.heading - (meta.whitespace.newline.markdown)</string>
```

Comment:
Nice idea with the meta.whitespace scope. Why the parentheses around meta.whitespace.newline.markdown?

Reply:
It's @keith-hall's idea really. I only used excludes like this a few times before, never would've thought to use it like this. And apparently it's fine without the parentheses.